### PR TITLE
DOCS-2083 Azure US3 Portal

### PR DESCRIPTION
**Pull request by ruthnaebeck**
### What does this PR do?
- Add documentation for Azure US3 Portal

### Motivation
Jira request from PM

### Preview
- [Azure integration page](https://docs-staging.datadoghq.com/ruth/docs-2083/integrations/azure)
- [Datadog in the Azure Portal](https://docs-staging.datadoghq.com/ruth/docs-2083/integrations/guide/azure-portal)

### Additional Notes
https://github.com/DataDog/dogweb/pull/62430

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: local/bin/py/build/configurations/pull_config_preview.yaml [hunk 1]
```diff
@@ -49,7 +49,7 @@
     contents:
 
     - action: integrations
-      branch: prod
+      branch: ruth/docs-2083
```
Comment (ruthnaebeck) [inline local/bin/py/build/configurations/pull_config_preview.yaml:52]:
```suggestion
      branch: prod
```
This is a temporary change for the preview site.

File: content/en/integrations/guide/azure-portal.md [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+    text: "Microsoft partnership embeds Datadog natively in the Azure portal"
+---
+
+{{< site-region region="us,eu,gov" >}}
+<div class="alert alert-warning">
+  This guide is for Datadog customers using US3 with Azure.
+</div>
+{{< /site-region >}}
+
+This guide is for managing the integration between Azure and Datadog in the Azure portal using the Datadog resource. The Datadog resource in Azure represents the connection between a Datadog organization and an Azure subscription. Create a Datadog resource in Azure before proceeding with this guide.
```
Comment (steveharrington1) [inline content/en/integrations/guide/azure-portal.md:19]: Let's add a link to the configuration section of the Azure integration page here where it says "create a datadog resource in Azure"

File: content/en/integrations/guide/azure-portal.md [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+Select "Change Plan" on the overview page to change your Datadog billing plan.
+
+{{< img src="integrations/guide/azure_portal/change-plan1.png" alt="Azure US3 Change plan" responsive="true" style="width:100%;">}}
+
+The portal retrieves all the available Datadog plans for your tenant, this includes any private offers. Select the appropriate plan and click "Change Plan".
+
+## Datadog org configurations
+### Metrics and logs
+
+Select "Metrics and logs" in the left sidebar to change the configuration rules for metrics and logs. See the Azure documentation to [Configure metrics and logs][1].
```
Comment (steveharrington1) [inline content/en/integrations/guide/azure-portal.md:96]: Let's link to our own docs for this

Comment (steveharrington1) [reply content/en/integrations/guide/azure-portal.md:96]: On the new azure integration main page

File: content/en/integrations/guide/azure-portal.md [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+
+To install the Datadog extension, select the appropriate app, then click "Install Extension". The portal asks for confirmation to install the extension. Select "OK" to begin installation. This restarts your app and adds the following settings:
+
+- `DD_API_KEY:<DEFAULT_API_KEY>`
+- `DD_SITE:us3.datadoghq.com`
+- `DD_LOGS_INJECTION:true`
+
+Azure shows the status as `Installing` until the Agent is installed and provisioned. After the Datadog Agent is installed, the status changes to `Installed`.
+
+**Note**: Ensure you are adding the extension to apps with supported runtimes. The Datadog resource does not limit or filter the list of apps.
```
Comment (steveharrington1) [inline content/en/integrations/guide/azure-portal.md:164]: let's link "supported runtimes" to https://docs.datadoghq.com/serverless/azure_app_services/#requirements

File: content/en/integrations/guide/azure-portal.md [hunk 5]
```diff
@@ -0,0 +1,195 @@
...
+| Resource name     | The app name                                                                                                                                                                 |
+| Resource status   | Whether the app service is stopped or running. The app service must be running to initiate install. If the app service is stopped, installing the Datadog Agent is disabled. |
+| App service plan  | The specific plan configured for the app service                                                                                                                             |
+| Extension version | The Datadog extension version number                                                                                                                                         |
+
+#### Install
+
+To install the Datadog extension, select the appropriate app, then click "Install Extension". The portal asks for confirmation to install the extension. Select "OK" to begin installation. This restarts your app and adds the following settings:
```
Comment (steveharrington1) [inline content/en/integrations/guide/azure-portal.md:156]: Let's link "Datadog extension" to https://docs.datadoghq.com/serverless/azure_app_services

File: content/en/integrations/guide/azure-portal.md [hunk 6]
```diff
@@ -0,0 +1,198 @@
...
+    text: "Microsoft partnership embeds Datadog natively in the Azure portal"
+---
+
+{{< site-region region="us,eu,gov" >}}
+<div class="alert alert-warning">
+  This guide is for Datadog customers using US3 with Azure.
+</div>
+{{< /site-region >}}
+
+This guide is for managing the integration between Azure and Datadog in the Azure portal using the Datadog resource. The Datadog resource in Azure represents the connection between a Datadog organization and an Azure subscription. [Create a Datadog resource][1] in Azure before proceeding with this guide.
```
Comment (steveharrington1) [inline content/en/integrations/guide/azure-portal.md:19]: @ruthnaebeck The "Create a Datadog Resource" link should direct to the azure marketplace page here: https://docs-staging.datadoghq.com/ruth/docs-2083/integrations/guide/azure-marketplace/?tab=create#create-datadog-resource

currently its sending to the main integration page which is no good.

File: content/en/integrations/guide/azure-portal.md [hunk 7]
```diff
@@ -0,0 +1,199 @@
...
+| Logs to Datadog    | The count of resources sending logs to Datadog through the integration    |
+| Metrics to Datadog | The count of resources sending metrics to Datadog through the integration |
+
+### Disable
+
+To stop sending logs and metrics from Azure to Datadog, select "Disable" on the overview page, then click "OK".
+
+{{< img src="integrations/guide/azure_portal/disable.png" alt="Azure US3 Disable integration" responsive="true" style="width:100%;">}}
+
+**Note**: After disabling, you are still billed for other Datadog services that aren't related to monitoring metrics and logs.
```
Comment (ruthnaebeck) [inline content/en/integrations/guide/azure-portal.md:65]:
```suggestion
**Note**: Disabling the Datadog resource stops the submission of metrics and platform logs to Datadog for the associated subscription. Any resources in the subscription submitting data directly to Datadog through the Agent or extension are unaffected.
```

**Review by steveharrington1 — APPROVED** [review]

**Pull request merged**